compile/Make_gcc.mak

```diff
@@ -14,7 +14,7 @@ libmigemo_DSO	= libmigemo.so.1
 libmigemo	= libmigemo.so
 EXEEXT		=
 CFLAGS_MIGEMO	= -fPIC
-LDFLAGS_MIGEMO	= -Wl,-rpath,.,-rpath,/usr/local/lib,-rpath,/usr/lib
+LDFLAGS_MIGEMO	= -Wl,-rpath,$(libdir),-rpath,.,-rpath,/usr/local/lib,-rpath,/usr/lib
```

Comment:
You should replace this like this

    +LDFLAGS_MIGEMO	= -Wl,-rpath,.,-rpath,$(libdir),-rpath,/usr/lib

Because your suggestion has problems:

1.   `rpath` order has been changed,
2.   same path is appear twice:

For example, `rpath` order with non-customized config results like this:

*   Original: `.`, `/usr/local/lib`, `/usr/lib`
*   Yours: `/usr/local/lib`, `.`, `/usr/local/lib`, `/usr/lib`

One of most important things is "your modification should not change anything for who doesn't need it"